Pod/Classes/Layouts/DisplaySwitchLayout.swift

```diff
@@ -17,12 +17,11 @@ import UIKit
 open class DisplaySwitchLayout: UICollectionViewLayout {
     
     fileprivate let numberOfColumns: Int
-    fileprivate let cellHeightPadding: CGFloat
-    fileprivate let cellWidthPadding: CGFloat
+    fileprivate let cellPadding: CGPoint
     fileprivate let staticCellHeight: CGFloat
     fileprivate let nextLayoutStaticCellHeight: CGFloat
     fileprivate var previousContentOffset: NSValue?
-    let layoutState: LayoutState
+    public let layoutState: LayoutState
```

Comment:
unclear why you're making it public. looks like project-dependant case

Reply:
Sorry, but what's problem with this?
1. The ```layoutState``` property is still read-only.
2. We already have it ```public``` in ```DisplaySwitchLayoutAttributes``` for use in ```UICollectionViewCell.apply``` method. Why not we have it ```public``` in ```DisplaySwitchLayout``` itself?
3. Also the user is the one who provides ```layoutState``` in ```DisplaySwitchLayout``` constructor (init).

In my case I need to access ```DisplaySwitchLayout.layoutState``` in ```UICollectionViewCell.willTransition```. I use it for cell setup to do some content updates before transition starts.